src/components/layout/FeaturedDomains/FeaturedDomains.tsx

```diff
@@ -0,0 +1,18 @@
+import { ArnsCard } from '../../cards';
+import './styles.css';
+
+function FeaturedDomains() {
+  return (
+    <div className="featuredDomains">
+      <span className="sectionHeader">Featured Domains</span>
+      <div className="cardContainer">
+        <ArnsCard name="ardrive" />
+        <ArnsCard name="arwiki" />
+        <ArnsCard name="pages" />
+      </div>
+      <span className="buttonLink">see more</span>
+    </div>
+  );
+}
```

Comment:
let's pass an array to this component so we can make it dynamic

```suggestion
import { useState } from 'react';
import { ArnsCard } from '../../cards';
import './styles.css';

function FeaturedDomains(domains: { name: string, txId: string}[]) {
  const DEFAULT_INCREMENT = 3;
  const [displayCount, setDisplayCount] = useState(DEFAULT_INCREMENT);

  function showMore(e: any){
    e.preventDefault();
    setDisplayCount(displayCount + DEFAULT_INCREMENT);
  }

  return (
    <div className="featuredDomains">
      <span className="sectionHeader">Featured Domains</span>
      <div className="cardContainer">
        {
          domains.map((domain, index) => {
            if (index > displayCount ) return;
            return <ArnsCard name={domain.name} txId={domain.txId}/>
          })
        }
      <button className="link" onClick={showMore}>see more</button>
    </div>
    </div>
  );
}

export default FeaturedDomains;

```